serpent/input_controllers/pyautogui_input_controller.py

```diff
@@ -9,17 +9,136 @@
+    KeyboardKey.KEY_T.name: "t",
+    KeyboardKey.KEY_Y.name: "y",
+    KeyboardKey.KEY_U.name: "u",
+    KeyboardKey.KEY_I.name: "i",
+    KeyboardKey.KEY_O.name: "o",
+    KeyboardKey.KEY_P.name: "p",
+    KeyboardKey.KEY_LEFT_BRACKET.name: "[",
+    KeyboardKey.KEY_RIGHT_BRACKET.name: "]",
+    KeyboardKey.KEY_BACKSLASH.name: "\\",
+    KeyboardKey.KEY_DELETE.name: "delete",
+    KeyboardKey.KEY_END.name: "end",
+    KeyboardKey.KEY_PAGE_DOWN.name: "pagedown",
+    KeyboardKey.KEY_NUMPAD_7.name: "num7",
+    KeyboardKey.KEY_NUMPAD_8.name: "num8",
+    KeyboardKey.KEY_NUMPAD_9.name: "num9",
+    KeyboardKey.KEY_NUMPAD_ADD.name: "add",
+    KeyboardKey.KEY_NUMPAD_PLUS.name: "add",
+    KeyboardKey.KEY_CAPSLOCK.name: "capslock",
+    KeyboardKey.KEY_A.name: "a",
+    KeyboardKey.KEY_S.name: "s",
+    KeyboardKey.KEY_D.name: "d",
+    KeyboardKey.KEY_F.name: "f",
+    KeyboardKey.KEY_G.name: "g",
+    KeyboardKey.KEY_H.name: "h",
+    KeyboardKey.KEY_J.name: "j",
+    KeyboardKey.KEY_K.name: "k",
+    KeyboardKey.KEY_L.name: "l",
+    KeyboardKey.KEY_SEMICOLON.name: ';',
+    KeyboardKey.KEY_APOSTROPHE.name: "'",
+    KeyboardKey.KEY_RETURN.name: "enter",
+    KeyboardKey.KEY_ENTER.name: "enter",
+    KeyboardKey.KEY_NUMPAD_4.name: "num4",
+    KeyboardKey.KEY_NUMPAD_5.name: "num5",
+    KeyboardKey.KEY_NUMPAD_6.name: "num6",
+    KeyboardKey.KEY_LEFT_SHIFT.name: "shiftleft",
+    KeyboardKey.KEY_Z.name: "z",
+    KeyboardKey.KEY_X.name: "x",
+    KeyboardKey.KEY_C.name: "c",
+    KeyboardKey.KEY_V.name: "v",
+    KeyboardKey.KEY_B.name: "b",
+    KeyboardKey.KEY_N.name: "n",
+    KeyboardKey.KEY_M.name: "m",
+    KeyboardKey.KEY_COMMA.name: ",",
+    KeyboardKey.KEY_PERIOD.name: ".",
+    KeyboardKey.KEY_SLASH.name: "/",
+    KeyboardKey.KEY_RIGHT_SHIFT.name: "shiftright",
+    KeyboardKey.KEY_UP.name: "up",
+    KeyboardKey.KEY_NUMPAD_1.name: "num1",
+    KeyboardKey.KEY_NUMPAD_2.name: "num2",
+    KeyboardKey.KEY_NUMPAD_3.name: "num3",
+    KeyboardKey.KEY_NUMPAD_RETURN.name: "separator",
+    KeyboardKey.KEY_NUMPAD_ENTER.name: "separator",
+    KeyboardKey.KEY_LEFT_CTRL.name: "ctrlleft",
+    KeyboardKey.KEY_LEFT_SUPER.name: "winleft",
+    KeyboardKey.KEY_LEFT_WINDOWS.name: "winleft",
+    KeyboardKey.KEY_LEFT_ALT.name: "altleft",
+    KeyboardKey.KEY_SPACE.name: "space",
+    KeyboardKey.KEY_RIGHT_ALT.name: "altright",
+    KeyboardKey.KEY_RIGHT_SUPER.name: "winright",
+    KeyboardKey.KEY_RIGHT_WINDOWS.name: "winright",
+    KeyboardKey.KEY_APP_MENU.name: "apps",
+    KeyboardKey.KEY_RIGHT_CTRL.name: "ctrlright",
+    KeyboardKey.KEY_LEFT.name: "left",
+    KeyboardKey.KEY_DOWN.name: "down",
+    KeyboardKey.KEY_RIGHT.name: "right",
+    KeyboardKey.KEY_NUMPAD_0.name: "num0",
+    KeyboardKey.KEY_NUMPAD_DECIMAL.name: "decimal",
+    KeyboardKey.KEY_NUMPAD_PERIOD.name: "decimal"
+}
+
+mouse_button_mapping = {
+    MouseButton.LEFT.name: "left",
+    MouseButton.MIDDLE.name: "left",
+    MouseButton.RIGHT.name: "left",
```

Comment:
I'm only on mobile, so I might have missed something in the diff, but three times left doesn't look right?